File: Modul-4/soal-5/PRAK405_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    int baris, kelipatan, total = 0;
    scanf("%d %d", &baris, &kelipatan);
    for (int i = 1; i <= baris; i++) {
        int total_kelipatan = 0;
        for (int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, kelipatan);
            total_kelipatan += j * kelipatan;
            total += j * kelipatan;
            if (j > 1) printf(" + ");
        }
        printf(" = %d\n", total_kelipatan);
    }
        printf("%d\n", total);
    return 0;
}

File: Modul-4/soal-5/PRAK405_2410817210017_OrlandoSugian.py
inputan = input("")
baris, kelipatan = map(int,inputan.split())
total = 0
for i in range(1, baris + 1):
    baris_hasil = ""  
    baris_total = 0  
    for j in range(i, 0, -1):
        if j != i:  
            baris_hasil += " + "
        baris_hasil += f"({j} * {kelipatan})"
        baris_total += j * kelipatan  
        total += j * kelipatan  
    print(baris_hasil, f"= {baris_total}")  
print(total)